docs/docs/guides/labs/dg/incrementally-adopting-dg/migrating-project.md

```diff
@@ -13,69 +13,93 @@ This guide is only relevant if you are starting from an _existing_ Dagster proje
 
 :::
 
-Let's begin with a Dagster project that has some assets:
+Let's begin with a basic existing Dagster project. Our current working
```

Comment:
I would just frame this "You have an existing project and use `pip`" then assume for the guide.